README.md

```diff
@@ -703,9 +703,6 @@ Tatusova, et al. 2008. “The Minimum Information about a Genome Sequence
 “GSC MIxS: Host-associatedMIMS.”
 <https://genomicsstandardsconsortium.github.io/mixs/Host-associatedMIMS/>.
 
-“GSC MIxS: Human-associatedMIMS.”
```

Comment:
Is there any reason why you removed this URL @pojith?

Reply:
no